api-inference-community/docker_images/adapter_transformers/requirements.txt

```diff
@@ -0,0 +1,3 @@
+starlette==0.14.2
```

Comment:
I think we also need torch here (see failing docker test)

Reply:
Thanks :)